File: Programmers/Lv.2/H-Index/42747.js
function solution(citations) {
    let answer = 0;
    
    citations.sort((a, b) => a - b);
    
    for (let i = 0; i < citations.length; i++) {
        const h = citations[i];
        const count = citations.length - i;
        if (h >= count) {
            answer = count;
            break;
        }
    }
    
    return answer;
}

File: Programmers/Lv.2/H-Index/42747.java
import java.util.*;

class Solution {
    public int solution(int[] citations) {
        int answer = 0;
        
        Arrays.sort(citations);
        
        for (int i = citations.length - 1; i >= 0; i--) {
            int h = citations[i];
            int count = citations.length - i;
            if (h >= count) {
                answer = count;
            }
        }
        
        return answer;
    }
}
